pandas/tseries/tests/test_tslib.py

```diff
@@ -688,6 +688,28 @@ def test_parsers_timezone_minute_offsets_roundtrip(self):
             converted_time = dt_time.tz_localize('UTC').tz_convert(tz)
             self.assertEqual(dt_string_repr, repr(converted_time))
 
+    def test_parsers_iso8601(self):
+        # GH 12060
```

Comment:
give a comment about what we are testing & doing here (and that all seps must be the same)